.gitignore

```diff
@@ -8,3 +8,8 @@
 package-lock.json
 pyvenv.cfg
 /_build
+*~
+lib64
```

Comment:
I do not see where either of these are generated. Either please explain or put into `~/.gitignore_global` and remove.

Reply:
I have them in my .gitignore_global, for some reason they get ignored. I will remove them.

Reply:
Fixed in 48b5ab20019214715481ba3e7f3dbc5f27affe22